RocketChat/Rocket.Chat.ReactNative
# [FIX] Create team crashing the app

## Proposed changes

Firebase's logEvent crashes on production if an argument is a boolean.
It can be seen in production on create team.
This PR converts the args to string, so they get logged and logEvent doesn't complain about it.

## Issue(s)	

- Create team is crashing the app

## How to test or reproduce

- run PR on debug and check if create team is getting logged on Firebase DebugView
- run PR on production and check if create team doesn't crash

## Screenshots

## Types of changes




## Checklist



## Further comments
